### PR TITLE
[zh-cn]: update the translation of `class` statement section

### PR DESCRIPTION
**Pull request by T34-active**
### Related issues and pull requests
https://github.com/mdn/translated-content/issues/18266

### CODE REVIEW

File: files/zh-cn/web/javascript/reference/statements/class/index.md [hunk 1]
```diff
@@ -1,73 +1,82 @@
...
 title: class
 slug: Web/JavaScript/Reference/Statements/class
+l10n:
+  sourceCommit: 4f86aad2b0b66c0d2041354ec81400c574ab56ca
 ---
 
 {{jsSidebar("Statements")}}
 
-**class 声明**创建一个基于原型继承的具有给定名称的新类。
+**`class`** 声明将新的 [类](/zh-CN/docs/Web/JavaScript/Reference/Classes){{Glossary("binding", "绑定")}} 到指定的名称。
```
Comment (skyclouds2001) [inline files/zh-cn/web/javascript/reference/statements/class/index.md:10]:
```suggestion
**`class`** 声明将新的[类](/zh-CN/docs/Web/JavaScript/Reference/Classes){{Glossary("binding", "绑定")}}到指定的名称。
```

File: files/zh-cn/web/javascript/reference/statements/class/index.md [hunk 2]
```diff
@@ -1,73 +1,82 @@
...
 
-{{EmbedInteractiveExample("pages/js/statement-class.html")}}
+你也可以使用[`class` 表达式](/zh-CN/docs/Web/JavaScript/Reference/Operators/class)来定义类。
```
Comment (skyclouds2001) [inline files/zh-cn/web/javascript/reference/statements/class/index.md:12]:
```suggestion
你也可以使用 [`class` 表达式](/zh-CN/docs/Web/JavaScript/Reference/Operators/class)来定义类。
```

**Review by JasonLamv-t — CHANGES_REQUESTED** [review]
Let's make this better.

File: files/zh-cn/web/javascript/reference/statements/class/index.md [hunk 3]
```diff
@@ -1,73 +1,82 @@
...
 ## 描述
 
-和类表达式一样，类声明体在[严格模式](/zh-CN/docs/Web/JavaScript/Reference/Strict_mode)下运行。构造函数是可选的。
+类声明的类主体在[严格模式](/zh-CN/docs/Web/JavaScript/Reference/Strict_mode)下执行。`class` 声明与 {{jsxref("Statements/let", "let")}} 声明非常相似：
 
-类声明不可以提升（这与[函数声明](/zh-CN/docs/Web/JavaScript/Reference/Statements/function)不同）。
+- `class` 声明的范围不仅限于函数，还包括代码块。
+- `class` 声明只能在其声明位置之后才能访问（参考[暂时性死区](/zh-CN/docs/Web/JavaScript/Reference/Statements/let#暂时性死区)）。因此 `class` 声明通常被认为是不可[变量提升](/zh-CN/docs/Glossary/Hoisting)的（与[函数声明](/zh-CN/docs/Web/JavaScript/Reference/Statements/function)不同）。
+- `class` 声明在脚本顶层声明时不会在 {{jsxref("globalThis")}} 上创建属性，（与[函数声明](/zh-CN/docs/Web/JavaScript/Reference/Statements/function)不同）。
+- `class` 在相同作用域内，任何其他声明都不能[重复声明](/zh-CN/docs/Web/JavaScript/Reference/Statements/let#重复声明)。
```
[the remark this answers is not in the document]
Comment (T34-active) [reply files/zh-cn/web/javascript/reference/statements/class/index.md:34]: Your translations are always refreshing to me👍

**Review by JasonLamv-t — APPROVED** [review]
Well done! Thank you! @T34-active

**Pull request merged**